src/Debugger.php

```diff
@@ -98,7 +98,10 @@ private function isRequestIgnored(ServerRequestInterface $request): bool
                 return true;
             }
         }
-        return false;
+        return (bool) ($request->hasHeader('X-Debug-Ignore') && $request->getHeaderLine('X-Debug-Ignore') === 'true')
```

Comment:
Should we use Yii prefix? 

`X-Yii-Debug-Ignore`

Reply:
Nope

Reply:
We also don't use the prefix in the headers middleware
https://github.com/yiisoft/yii-debug-api/blob/master/src/Middleware/DebugHeaders.php#L30